src/travel-aid/components/StopSignalButton.tsx

```diff
@@ -54,6 +54,7 @@ export const StopSignalButton = ({
         <MessageInfoBox
           type="error"
           message={t(dictionary.genericErrorMsg)}
+          a11yAnnounce
```

Comment:
Huh, don't know why that is 🤔 Haven't gotten any issues like that while myself. But it might make the most sense to use assertive here, so I'm all for making it configurable.